src/ui/ReactionButton/index.tsx

```diff
@@ -18,7 +21,7 @@ export interface ReactionButtonProps {
     e: MouseEvent<HTMLDivElement> | KeyboardEvent<HTMLDivElement> | TouchEvent<HTMLDivElement>
   ) => void;
 }
-const ReactionButton = React.forwardRef((props: ReactionButtonProps, ref: RefObject<HTMLDivElement>): ReactElement => {
+const ReactionButton: ForwardRefExoticComponent<ReactionButtonProps> = React.forwardRef((props: ReactionButtonProps, ref: ForwardedRef<HTMLDivElement>) => {
```

Comment:
ForwardRefExoticComponent 😮

Reply:
nit: the line is too long.. I thought we set 120 as limit in eslint

Reply:
We can change the max line num too ;)